airbyte-integrations/connectors/source-tiktok-marketing/integration_tests/spec.json

```diff
@@ -41,36 +41,7 @@
             },
             "required": ["app_id", "secret", "access_token"]
           }, {
-            "title": "Production Access Token",
```

Comment:
@sherifnada 

'Production' - is completely removed. 

Note: In order to pass all tests I have to update 5 config files (replace "prod_access_token" to "oauth2.0")